Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'http://rubygems.org'
 
-ruby '2.1.1'
+ruby '2.1.7'
```

Comment:
A couple reasons: Firstly, there was a bug in the Dockerfile for `ruby:2.1.1` that caused it to bring a up 2.1.2 container (see phuongnd08's comment here: https://hub.docker.com/r/library/ruby/tags/). Also, they made some optimizations to the latest `2.1.7` branch that brought the total image size way down (from 611MB to 260MB). I'd prefer to avoid bumping the Ruby version simply to add Docker support, but in this case the only other option was to fork the Docker-supported version. I'll fix the `.ruby-version`, though. Missed that.